### ISSUE TITLE
Provision failed with SPServiceApplication Managed Metadata Service Application in non-enUS

### ISSUE THREAD

**Issue by luigilink** [PowerShell/SharePointDsc#417]
**Details of the scenario you try and problem that is occurring:**
Provision failed with SPServiceApplication Managed Metadata Service Application in non-enUS.
The propertie TypeName does't work because it returns french name instead of "Managed Metadata Service"
Add GetType().FullName in FilterScript parameter for Where-Object Function:

```
        $serviceApp = $serviceApps | Where-Object -FilterScript {
            $_.TypeName -eq "Managed Metadata Service" -or `
            $_.GetType().FullName -eq "Microsoft.SharePoint.Taxonomy.MetadataWebServiceApplication"
        }
```

**The DSC configuration that is using the resource:**

```
SPManagedMetaDataServiceApp APPLICATION_SpsSvcAppManagedMetadataServiceApp
{  
            DependsOn            = '[SPServiceAppPool]APPLICATION_SpsSvcAppMainServiceAppPool','[SPServiceInstance]APPLICATION_SpsSvcAppManagedMetadataServiceInstance'
            PsDscRunAsCredential = $InstallAccount
            Name                 = "***"
            ProxyName            = "***"
            ApplicationPool      = "***"
            DatabaseServer       = "***"
            DatabaseName         = "***"
        }
```

**Version of the Operating System and PowerShell the DSC Target Node is running:**
- PowerShell Version: 5.0.10586.117
- OS Version : Windows 2012 R2 + French Language Pack

**Version of the DSC module you're using:** SharePointDSC 1.3.0.0

**Comment by luigilink** [comment]
Fix suggested in PR #418

**Comment by BrianFarnhill** [comment]
We will have this issue in most of our service apps I would think. I'll review your fix and the other resources tomorrow to confirm.

**Comment by luigilink** [comment]
Yes I confirm, what do you prefer:
One PR by Service Apps
Or One PR for All Service Apps

**Comment by BrianFarnhill** [comment]
One for All of them would be easier to review and manage

**Comment by luigilink** [comment]
Fix suggested in PR #421

**Comment by luigilink** [comment]
I forgot a few resrouces with the typeName property.
Fix suggested in PR #433

**Comment by ykuijs** [comment]
@BrianFarnhill The two mentioned PR's are merged, so this issue can be closed.
